src/main/java/org/molgenis/vcf/report/helpers/jackson/phenopacket/HtsFileMixin.java

```diff
@@ -0,0 +1,41 @@
+package org.molgenis.vcf.report.helpers.jackson.phenopacket;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+public interface HtsFileMixin  extends PhenopacketModelMixin {
```

Comment:
Can you explain what the function of the mixins is? Can't we just serialize value objects from the phenopackets dependency?

Reply:
nope, using the phonepacket classes generates a loop that crashes jackson 
+
 you get fields for all the getters stuff like: "get*Bytes" for all fields.

This is the part I meant the last few days when I said we need te review if we want to use the Phenopacket classes... or maybe make our own value objects based on them...

Reply:
The mixin allows you to ignore or rename fields in the classes